sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_polling.py

```diff
@@ -60,67 +93,32 @@ def _current_body(self):
         return {}
 
     @property
-    def operation_id(self) -> str:
+    def details(self) -> Mapping[str, Any]:
```

Comment:
We should call out the breaking change on the changelog, i.e. accessing metadata under details now